package.json

```diff
@@ -42,11 +42,9 @@
     "babel-plugin-detective": "2.0.0",
     "babel-plugin-module-resolver": "3.0.0",
     "babel-plugin-transform-async-to-generator": "6.24.1",
-    "babel-plugin-transform-runtime": "6.23.0",
-    "babel-polyfill": "6.26.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-polyfill": "^6.26.0",
```

Comment:
Use exact versions (`"6.26.0"`)

Reply:
I don't understand why you're doing that. yarn.lock exists for stability purpose and semver exists for easier upgrade.

Reply:
So we know at a glance what version we're actually using, updates are always done manually in package.json anyway. We've had a few problems with packages not respecting semver before.